src/background/stores/details-view-store.ts

```diff
@@ -106,16 +106,16 @@ export class DetailsViewStore extends PersistentStore<DetailsViewStoreData> {
         this.emitChanged();
```

Comment:
It doesn't look like this is including the async work from `emitChanged` in the returned store promises. Is that intentionally left for a different PR?

Reply:
Yes, once this is in I plan on updating https://github.com/microsoft/accessibility-insights-web/pull/5895 to await emitChanged :)